contracts/libs/Actions.sol

```diff
@@ -281,6 +295,25 @@ library Actions {
         return SettleVaultArgs({owner: _args.owner, vaultId: _args.vaultId, to: _args.secondAddress});
     }
 
+    function _parseLiquidateArgs(ActionArgs memory _args) internal pure returns (LiquidateArgs memory) {
+        require(_args.actionType == ActionType.Liquidate, "Actions: can only parse arguments for liquidate action");
+        require(_args.owner != address(0), "Actions: cannot liquidate vault for an invalid account owner");
+        require(_args.secondAddress != address(0), "Actions: cannot send collateral to an invalid account");
+        require(_args.data.length == 32, "Actions: cannot parse liquidate action with no round id");
+
```

Comment:
Do we also want a check to not allow self liquidation?

Reply:
We currently allow for self liquidation

Reply:
thoughts @aleone @zkoticha ?